.github/workflows/deploy-preview.yml

```
@@ -703,52 +698,176 @@
+          echo "🔍 Checking for existing Amplify app: ${AMPLIFY_APP_NAME}"
+
+          # Check if Amplify app already exists
+          EXISTING_APP_ID=$(aws amplify list-apps \
+            --region ${{ env.AWS_REGION }} \
+            --query "apps[?name=='${AMPLIFY_APP_NAME}'].appId" \
+            --output text 2>/dev/null || echo "")
+
+          if [[ -n "$EXISTING_APP_ID" && "$EXISTING_APP_ID" != "None" ]]; then
+            echo "✅ Found existing Amplify app with ID: $EXISTING_APP_ID"
+            echo "existing-amplify-app-id=$EXISTING_APP_ID" >> $GITHUB_OUTPUT
+            
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)